met-api/src/met_api/resources/__init__.py

```diff
@@ -21,6 +21,7 @@
 That are used to expose operational health information about the service, and meta information.
 """
 
+from re import A
```

Comment:
Remove this unused import

Reply:
Thanks @NatSquared Not sure why it was not detected in Flake or pylint